ga4gh/datamodel/ontologies.py

```diff
@@ -26,11 +29,28 @@ def add(self, id_, name):
         self._nameIdMap[id_] = name
         self._idNameMap[name] = id_
 
+    def getLocalId(self):
+        return self._localId
+
     def getId(self, name):
-        return self._idNameMap[name]
+        try:
+            id_ = self._idNameMap[name]
+        except KeyError:
+            raise exceptions.OntologyMapIdException
```

Comment:
What's the rationale for raising an OntologyMapIdException here  rather than a KeyError? There are advantages to making the Ontology object behave as much like a python map as possible, and raising a KeyError is a natural thing to do.